BoardModulePkg/Test/Mock/Library/GoogleTest/MockBiosIdLib/MockBiosIdLib.inf

```diff
@@ -0,0 +1,33 @@
+## @file MockBiosIdLib.inf
+# Google Test mock for BiosIdLib
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = MockBiosIdLib
+  FILE_GUID                      = FD03FA6D-7447-499C-B3AF-D89450F7739A
+  MODULE_TYPE                    = HOST_APPLICATION
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = BiosIdLib
+  PI_SPECIFICATION_VERSION       = 0x0001000A
+
+#
+#  VALID_ARCHITECTURES           = IA32 X64

```

Comment:
Is this really restricted to IA32 and X64?

Reply:
Because of its dependency on the Subhook submodule, the GoogleTest framework with gmock doesn't currently support ARM, as far as I know. For the mock infs themselves, not sure how important/applicable it is.

Reply:
I forgot about that. That is a valid reason to leave it as-is.